### PR TITLE
bump mapbox gl js to v3, replace Streets with Standard

### PR DESCRIPTION
**Pull request by chriswhong**
image

### CODE REVIEW

File: package-lock.json
Comment (OdNairy) [inline package-lock.json:1]: Is it expected that `npm ci` command fails without `--force` flag?

Comment (OdNairy) [reply package-lock.json:1]: CleanShot 2024-01-31 at 09 48 50@2x

**Review by OdNairy — APPROVED** [review]
Thanks @chriswhong for an update!

**Review by wboykinm — COMMENTED** [review]
It feels like a good sign for the dev experience if this is what's required to go from v2 to v3 and streets to standard. The modularity is really straightforward. 🤘

File: package-lock.json
Comment (chriswhong) [reply package-lock.json:1]: I remember the same gl-geocoder dependency issue popping up when v3 came out.  Yes, this is expected until we bump the version on mapbox-gl-geocoder.  I'll get to it soon along with some other dependencies.

**Pull request merged**